js/count-letter-in-string.js

```diff
@@ -0,0 +1,17 @@
+// https://www.codewars.com/kata/52efefcbcdf57161d4000091/train/javascript
+
+function count(string) {
+  const lettersArr = string.split('');
+  let lettersCount = {};
+
+   lettersArr.forEach((letter, i) => {
```

Comment:
Is the `i` index necessary? Can you not use `letter` whenever you need the value :slightly_smiling_face: ?